lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -1357,17 +1357,50 @@ namespace Js
 
         RecyclableObject* instance = RecyclableObject::FromVar(instanceVar);
         ScriptContext* scriptContext = instance->GetScriptContext();
+
+        if (!PHASE_OFF1(IsConcatSpreadableCachePhase))
+        {
+            BOOL retVal = FALSE;
+            Type *instanceType = instance->GetType();
+            IsConcatSpreadableCache *isConcatSpreadableCache = scriptContext->GetThreadContext()->GetIsConcatSpreadableCache();
+
+            if (isConcatSpreadableCache->TryGetIsConcatSpreadable(instanceType, &retVal))
+            {
+#ifdef ENABLE_DEBUG_CONFIG_OPTIONS
+                if (Js::Configuration::Global.flags.Trace.IsEnabled(Phase::IsConcatSpreadableCachePhase))
```

Comment:
nit : change this to OUTPUT_TRACE, that macro checks for IsEnabled as well also you don't need to put #ifdef also.